doc/nohelp.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>No help found</title>
+  <meta charset="UTF-8">
+</head>
+
+<body>
+<center><a href="http://qgis.org"><img src="qgis-icon-60x60.png" border=0></a></center>
+<h1><center>Oops!</center></h1>
+<p>Sorry, help is not available. Probably you have no Internet connection,
```

Comment:
What about something like:

```
<p>QGIS help is currently not available. This is likely because:</p>
<ul>
<li>No internet connection is available</li>
<li>The offline help files are not available or the location of these files is not properly configured</li>
<li>No help content is available for the requested topic</li>
</ul>
<p>Please check that the correct location of the offline help files is specified in the QGIS options dialog (Settings&nbsp;&rarr; Options&nbsp;&rarr;System).</p>
```